docs/sphinx/features/device_allocators.rst

```diff
@@ -28,17 +28,18 @@ exceed the size that was used when making the device allocator.
 Retrieving a DeviceAllocator Object
 -----------------------------------
 
-The :class:`umpire::get_device_allocator` host/device function returns the DeviceAllocator object that corresponds 
+After creating a DeviceAllocator, we can immediately start using that allocator to allocate device memory. To do this, we
+have the :class:`umpire::get_device_allocator` host/device function which returns the DeviceAllocator object corresponding 
```

Comment:
I think that this example should be replaced with just an example that uses the name as this will likely be the most common case.  If an example with IDs is necessary, the ID should be passed in (from somewhere) as a parameter and not be a integer literal.